content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -468,9 +529,9 @@ For get and list, the semantics of resource version are:
 
 **List:**
 
-v1.19+ API servers support the `resourceVersionMatch` parameter, which
-determines how resourceVersion is applied to list calls.  It is highly
-recommended that `resourceVersionMatch` be set for list calls where
+v1.19+ API servers also support the `resourceVersionMatch` parameter, which
+determines how resourceVersion is applied to List calls.  It is highly
```

Comment:
nit: `resourceVersion`? OR "how the version is applied to `list` requests"?
nit: It is recommended that ...